wandb/apis/public.py

```diff
@@ -2455,6 +2457,10 @@ def url(self):
     def name(self):
         return self.config.get("name") or self.id
 
+    @property
+    def state(self):
+        return self._state
```

Comment:
Classes that inherit from `Attrs` will magically have all `._attrs` keys exposed as properties.  We should be able to get rid of the `_state` instance variable and the property.  It should magically Just Work™